engine/src/main/resources/templates/SubscriptionWatch/usageThresholdExceededEmailBodyV2.html

```diff
@@ -2,14 +2,47 @@
 {@boolean renderButtonSection1=false}
 {#include Common/insightsEmailBody}
 {#content-title}
-    Subscription Watch - Subscription Services
+    Subscription Watch | Subscription Services
```

Comment:
"Subscriptions Usage" is the name of the service.

Subscription Watch is a very old name, and everyone still uses it internally, but that is not the customer-facing name in the Hybrid Cloud Console.